core/chain_util.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	blockHashPre  = []byte("block-hash-")
 	blockNumPre   = []byte("block-num-")
-	expDiffPeriod = big.NewInt(100000)
+	ExpDiffPeriod = big.NewInt(100000)
```

Comment:
This should **not** be used for the olympic net because it will break consensus when importing the olympic chain failing on difficulty.